README.md

```diff
@@ -65,4 +65,27 @@
 
 2. To run multiple tags(tiers) in a single run, provide tag names appended in below format - 
 
-    `npx cypress run --env grepTags=@tag1+@tag2`
\ No newline at end of file
+    `npx cypress run --env grepTags=@tag1+@tag2`
+
+#### Running tests locally over Tackle UI hosted on minikube
+
+1. Install minikube on your local machine by referring to [docs](https://minikube.sigs.k8s.io/docs/start/)
+2. Start minikube by specifying needed addons and kubernetes version using below command
+
+    `minikube start --addons=registry --addons=ingress --kubernetes-version v1.20.2`
+
+3. Create new namespace for deploying Tackle UI
+
+    `kubectl create ns $tackleNamespace`
+
+4. Save the yaml from tackle [repo](https://github.com/konveyor/tackle/blob/main/kubernetes/kubernetes-tackle.yaml) locally, eg - /tmp/tackle/tackle.yaml
+
+    `kubectl create -f /tmp/tackle/tackle.yaml -n $tackleNamespace`
+
+5. Wait for some time to let deployment finish and extract the Tackle UI web page url (typically an IP) using command - 
```

Comment:
@nitishSr 
I think it will be good if we specify these steps too.
```

      - name: Verify keycloak
        run: kubectl wait deployment/keycloak --for condition=available --timeout=-1s -n tackle
      - name: Verify tackle-controls
        run: kubectl wait deployment/tackle-controls --for condition=available --timeout=-1s -n tackle
      - name: Verify tackle-application-inventory
        run: kubectl wait deployment/tackle-application-inventory --for condition=available --timeout=-1s -n tackle
      - name: Verify tackle-pathfinder
        run: kubectl wait deployment/tackle-pathfinder --for condition=available --timeout=-1s -n tackle
      - name: Verify tackle-ui
        run: kubectl wait deployment/tackle-ui --for condition=available --timeout=-1s -n tackle
```

Ref: https://github.com/konveyor/tackle-ui-tests/blob/main/.github/workflows/k8s-ci.yml#L38

Reply:
@ganeshhubale these steps would be helpful for debugging or knowing if any of the pod didn't start, that may lead to Tackle UI not running. But, does not fit to be provided in the process to start it (the intention of these steps in PR). We may add these steps as separate section for debugging. But again, it will then add up more info, which might not be that relevant in our context.